app/models/miq_expression.rb

```diff
@@ -1775,15 +1775,9 @@ def self.atom_error(field, operator, value)
                                                                                     :error_message => err.message}
       end
       return false
-    when :ruby
-      return _("Ruby Script must not be blank") if value.blank?
-      return false
```

Comment:
Why would a PR that is adding tests for MiqExpression drop support for this?  You should maintain the same compatibility in this PR and open a new PR to drop support for this if that's desired.